tool/jt.rb

```diff
@@ -792,28 +793,38 @@ def pr_push(*args)
         "github/pr/#{pr_number}"
       end
 
-      sh 'git', 'push', '--force', '--no-verify', bb, "#{github_pr_branch}:refs/heads/#{target_branch}"
+      sh 'git', 'push', '--force', '--no-verify', Remotes.bitbucket, "#{github_pr_branch}:refs/heads/#{target_branch}"
     end
 
     def pr_update_master(skip_upstream_fetch: false)
-      sh 'git', 'fetch', upstream unless skip_upstream_fetch
-      sh 'git', 'push', '--no-verify', bb, "#{upstream}/master:master"
+      sh 'git', 'fetch', Remotes.github unless skip_upstream_fetch
+      sh 'git', 'push', '--no-verify', Remotes.bitbucket, "#{Remotes.github}/master:master"
     end
+  end
+
+  module Remotes
+    include ShellUtils
+    extend self
 
-    def bb
-      remote_urls.find { |r, u| u.include? 'ol-bitbucket' }.first
+    def bitbucket(dir = TRUFFLERUBY_DIR)
+      candidate = remote_urls(dir).find { |r, u| u.include? 'ol-bitbucket' }
+      candidate.first if candidate
     end
 
-    def upstream
-      remote_urls.find { |r, u| u.include? 'oracle/truffleruby' }.first
+    def github(dir = TRUFFLERUBY_DIR)
+      candidate = remote_urls(dir).find { |r, u| u.include? 'github.com' }
```

Comment:
This needs to be more precise, I have multiple truffleruby remotes on GitHub.
Maybe `git@github.com:oracle` || `https://github.com/oracle`?

Reply:
good catch